<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SoundPage</class>
 <widget class="QWidget" name="SoundPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>463</width>
    <height>370</height>
   </rect>
  </property>
  <layout class="QGridLayout">
   <item row="1" column="0">
    <widget class="QGroupBox" name="eventGroup">
     <layout class="QVBoxLayout" name="eventGroupVLayout">
      <item>
       <layout class="QGridLayout" name="eventDataGLayout">
        <item row="3" column="1">
         <layout class="QHBoxLayout" name="eventFilenameHLayout">
          <property name="spacing">
           <number>2</number>
          </property>
          <item>
           <widget class="LineEditClear" name="filenameEdit"/>
          </item>
         </layout>
        </item>
        <item row="1" column="1">
         <widget class="QLabel" name="eventName">
          <property name="text">
           <string notr="true">Event name</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="eventLabel">
          <property name="text">
           <string>Event:</string>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="fileNameLabel">
          <property name="text">
           <string>Filename:</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="buttonHLayout">
        <item>
         <spacer name="buttonHSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="defaultButton">
          <property name="text">
           <string>Default</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="browseButton">
          <property name="text">
           <string>Browse</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="playButton">
          <property name="icon">
           <iconset resource="../images.qrc">
            <normaloff>:/images/player_play.png</normaloff>:/images/player_play.png</iconset>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QTreeWidget" name="eventTreeWidget">
     <property name="editTriggers">
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
     <property name="allColumnsShowFocus">
      <bool>true</bool>
     </property>
     <column>
      <property name="text">
       <string notr="true">1</string>
      </property>
     </column>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>LineEditClear</class>
   <extends>QLineEdit</extends>
   <header>gui/common/LineEditClear.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>eventTreeWidget</tabstop>
  <tabstop>filenameEdit</tabstop>
  <tabstop>defaultButton</tabstop>
  <tabstop>browseButton</tabstop>
  <tabstop>playButton</tabstop>
 </tabstops>
 <resources>
  <include location="../images.qrc"/>
 </resources>
 <connections/>
</ui>
